AutomatedTesting/Gem/PythonTests/prefab/TestSuite_Main.py

```diff
@@ -31,7 +30,34 @@ def test_PrefabLevel_OpensLevelWithEntities(self, request, workspace, editor, la
         from . import PrefabLevel_OpensLevelWithEntities as test_module
         self._run_prefab_test(request, workspace, editor, test_module)
 
-    def test_PrefabLevel_BasicWorkflow(self, request, workspace, editor, launcher_platform):       
-        from . import PrefabLevel_BasicWorkflow as test_module
+
+@pytest.mark.SUITE_main
+@pytest.mark.parametrize("launcher_platform", ['windows_editor'])
+@pytest.mark.parametrize("project", ["AutomatedTesting"])
+class TestPrefabBasicWorkflows(TestAutomationBase):
```

Comment:
It's better to merge this into a single class. That way the AP can be reused on all tests and also will be parallizable in the future.

Reply:
Will fix it.

Reply:
Fixed.